.github/workflows/test.yml

```diff
@@ -45,6 +62,7 @@ jobs:
       run: |
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
+        pip install ckantoolkit
```

Comment:
ckantoolkit should be installed in dev-requirements.txt or requirements.txt if it's needed.

Reply:
thx